seeds/seedGenerator.js

```diff
@@ -0,0 +1,44 @@
+const { moviesData, commentsData } = require('./seedMovies');
+const movieService = require('../services/movieServices');
+const commentService = require('../services/commentServices');
+const { checkIfDataExists } = require('../utils/dbUtils');
+const env = process.env.NODE_ENV || 'development';
+const { sequelize } = require('../models')
+
+module.exports.seed = async () => {
+  if (env === 'test') {
```

Comment:
Why?? Thought we discussed not breaking these boundaries??

Reply:
Yes, removed